test_conformance/conversions/test_conversions.cpp

```diff
@@ -1141,421 +1210,444 @@ static int DoTest( cl_device_id device, Type outType, Type inType, SaturationMod
 
         // Copy the results to the device
         writeInputBuffer = NULL;
-        if( (error = clEnqueueWriteBuffer(gQueue, gInBuffer, CL_FALSE, 0, count * gTypeSizes[inType], gIn, 0, NULL, &writeInputBuffer )))
+        if ((error = clEnqueueWriteBuffer(gQueue, gInBuffer, CL_TRUE, 0,
+                                          count * gTypeSizes[inType], gIn, 0,
+                                          NULL, &writeInputBuffer)))
```

Comment:
I don't think the writeInputBuffer event is released anymore after these changes.

Since we're no longer using an event callback on this event can it be removed completely?

Reply:
Removed it.